File: ControlMedico.back-end/ControlMedico.Modelos/Modelos/Paciente.cs
using ControlMedico.Modelos.Util;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace ControlMedico.Modelos.Modelos
{
    public partial class Paciente
    {

        [Key]
        public int IdPaciente { get; set; }
        [Required]
        [StringLength(12)]
        public string Identificacion { get; set; }
        public EnumTipoIdentificacion TipoIdentificacion { get; set; }
        [Required]
        [StringLength(100)]
        public string NombreCompleto { get; set; }
        [Column(TypeName = "date")]
        public DateTime FechaNacimiento { get; set; }
        public EnumGenero Genero { get; set; }
        [Required]
        [StringLength(250)]
        public string Residencia { get; set; }
        [Required]
        [StringLength(8)]
        public string Telefono { get; set; }
        public string DesTipoIdentificacion { get { return ExtensionEnum.ObtenerDescripcionEnum((EnumTipoIdentificacion)TipoIdentificacion); } }
        public string DesGenero { get { return ExtensionEnum.ObtenerDescripcionEnum((EnumGenero)Genero); } }

        [InverseProperty("InfoPaciente")]
        public virtual ICollection<Cita> Cita { get; set; }
    }
}

File: ControlMedico.front-end/ControlMedicoAngular/src/app/_models/paciente.ts
import { EnumTipoIdentificacion, EnumGenero } from './enumerados';

export class Paciente {
        public IdPaciente: number;
        public Identificacion: string;
        public TipoIdentificacion: EnumTipoIdentificacion;
        public DesTipoIdentificacion: string;
        public NombreCompleto: string;
        public FechaNacimiento: Date;
        public Genero: EnumGenero;
        public DesGenero: string;
        public Residencia: string;
        public Telefono: string;
}